### PR TITLE
ENH: Add spherical CNN cortex parcellation paper review

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add spherical CNN cortex parcellation paper review.

### CODE REVIEW

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+layout: review
+title: "Cortical Surface Parcellation using SphericalConvolutional Neural
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: whitespace -> Spherical Convolutional

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Networks"
+tags: brain deep-learning medical segmentation
+author: "Jon Haitz Legarreta Gorroño"
+cite:
+    authors: "Prasanna Parvathaneni et al."
+    title:   "Cortical Surface Parcellation using SphericalConvolutional Neural
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: whitespace -> Spherical Convolutional

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    Networks"
+    venue:   "arXiv 2019"
+pdf: "https://arxiv.org/pdf/1907.05395.pdf"
+---
+
+
+# Highlights
+
+Authors present present a cortical surface parcellation methods using spherical
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: ... parcellation method ...

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+- Novel features optimized over cortical parcel boundaries.
+- Data augmentation driven by their intermediate deformation fields.
+
+Their method outperforms traditional multi-atlas and naive spherical U-Net
+approaches.
+
+
+# Introduction
+
+For regional-based morphological analysis, cortical surfaces need to be
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: region-based

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+consistently subdivided into multi-regions based on cortical parcellation
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: multiple regions ?

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+protocols. Consistent labeling of cortical regions is challenging due to the
+complicated cortical folds and inter-subject variability.
+
+Multi-atlas cortical parcellation approaches tend to provide better performance
+as the number of atlases increases. Unfortunately, inter-subject registration
+is unavoidable in this approach to align multiple atlases.
+
+Traditional CNNs architectures are still immature in handling non-uniform data
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: CNNs -> CNN

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+with high complexity due to the Euclidean space coherence incorporated with
+existing deep architectures.
+
+Spherical CNNs have recently emerged to deal with spherical domain data. Authors
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: The authors ...

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+propose a novel cortical parcellation approach using a deep spherical U-Net
+encoding surface mesh features.
+
+
+# Methods
+
+To compute the input features to the CNN, the following features are computed:
+- The **deformation field**: they use a spherical surface registration method
+that reconstructs the deformation field by a linear combination of spherical
+harmonics coefficients with degrees $$l=0 \cdot 10$$.
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: \cdot -> \ldots

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+- The **boundary map**: authors compute deformation fields that align parcel
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: the authors

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+
+To create a template, they co-register training samples in an iterative manner
+and compute a distance map of the mode (most frequent) cortical labels across
+the training set after their registration to the template using the three
+geometric features. They then register the normalized distance map to the
+template distance map to produce the deformation fields.
+
+![](/article/images/SphericalCNNCortexParcellation/Architecture.png)
+
+Authors used a spherical U-net architecture designed for segmentation tasks.
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: The authors ...

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+They provide the described geometric features to the input channels and the
+corresponding labels to the output channels.
+
+Data augmentation is performed by using all the deformations of the spherical
+harmonics between $$l=0 \cdot 10$$.
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: ... harmonics for $$l=0 \ldots 10$$.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+smooth parcel boundaries.
+
+## Data
+
+427 T1-weighted 3T MRI images.
+
+
+# Results
+
+The cortical surfaces and their spherical mapping were reconstructed via a
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: mappings ? (plural)

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+
+No regions were found with significantly reduced Dice overlap.
+
+![](/article/images/SphericalCNNCortexParcellation/Results_images.png)
+![](/article/images/SphericalCNNCortexParcellation/Results_tables.png)
+
+
+# Conclusions
+
+Authors presented a cortical parcellation method using spherical U-Net with
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: The auhtors present a ... using a spherical U-Net ...

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+novel features optimized over cortical parcellation boundaries.
+
+The proposed method achieved qualitatively and quantitatively better
```
Comment (djoerch) [inline article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: ... achieves ...

Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Done in  584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 13 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Done in  584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 1 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Done in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 3 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Done in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 2 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Fixed in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 4 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Changed in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 6 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Fixed in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 7 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Changed in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 8 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Fixed in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 9 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Changed in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 10 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Cahnged in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 12 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Fixed in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 11 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Fixed in 584ea4d.

File: article/_posts/2019-07-25-SphericalCNNCortexParcellation.md (hunk 5 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-25-SphericalCNNCortexParcellation.md]: Changed in 584ea4d.

**Comment by jhlegarreta** [comment]
Thanks @djoerch !

**Pull request merged**